### PR TITLE
Remove all the links to academy and help center - closes #2496

### LINKED ISSUE #2496: Remove all the links to academy and help center
### Description
We need to remove all the links to Lisk Academy and help center. 

component: PassphraseBackup
link: howToStorePassphrase

component: UnlockDevice
link: ledgerNanoSHelp

component: RecentTransactions
link: outgoingTransactions

component: FormBase
link: transactionFee

component: SelectName
link: votingAndDelegates

component: Login
link: whatIsAnPassphrase

### PR DESCRIPTION
**Pull request by osvaldovega**
…t tests

### What issue have I solved?

-- #2496 

### How have I implemented/fixed it?

This PR is for remove the following links from the application.

- [x] **component:** PassphraseBackup - **link:** howToStorePassphrase.
- [x] **component:** UnlockDevice - **link**: ledgerNanoSHelp.
- [x] **component:** RecentTransactions - **link**: outgoingTransactions.
- [x] **component:** FormBase - **link**: transactionFee.
- [x] **component:** SelectName - **link**: votingAndDelegates.
- [x] **component:** Login - **link**: whatIsAnPassphrase.

After update the components I had to update 2 unite tests files are those links are not part of the page any more.

### How has this been tested?

1- Run the app.
2- Check all the pages mentioned above and see that there is not link anymore in those specific pages.

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)

### CODE REVIEW

**Comment by slaweet** [comment]
The PR removes all links mentioned in the issue, it matches the requirements.

There is a problem with the requirements:
- :books: the requirements don't list all the links. 

IMO all these mentions should be removed as well:

```
$ git grep Academy
app/src/menu.js:            label: i18n.t('Lisk Academy'),
i18n/locales/de/common.json:  "Academy": "Academy",
i18n/locales/de/common.json:  "Lisk Academy": "Lisk Academy",
i18n/locales/de/common.json:  "The Lisk Academy is an entirely free, unbiased and comprehensive educational platform about blockchain technology, containing something for everyone, regardless of what level of knowledge you are at.": "Die Lisk Academy ist eine kostenlose, unparteiische und umfrangreiche Wissensplatform über Blockchaintechnologie, die, unabhängig vom eigenen Wissensstand, für jeden etwas beinhaltet.",
i18n/locales/de/common.json:  "Want to know more? We’ve got you covered. Read more about Lisk consensus algorithm and its benefits in the Lisk Academy.": "Möchest du mehr wissen? Wir geben dir Deckung. Weitere Informationen zum Lisk-Konsensus-Algorithmus und seinen Vorteilen findest du in der Lisk Academy.",
i18n/locales/de/common.json:  "What is Lisk Academy?": "Was ist die Lisk Academy?",
i18n/locales/en/common.json:  "Academy": "Academy",
i18n/locales/en/common.json:  "Lisk Academy": "Lisk Academy",
i18n/locales/en/common.json:  "The Lisk Academy is an entirely free, unbiased and comprehensive educational platform about blockchain technology, containing something for everyone, regardless of what level of knowledge you are at.": "The Lisk Academy is an entirely free, unbiased and comprehensive educational platform about blockchain technology, containing something for everyone, regardless of what level of knowledge you are at.",
i18n/locales/en/common.json:  "Want to know more? We’ve got you covered. Read more about Lisk’s consensus algorithm and its benefits in the Lisk Academy.": "Want to know more? We’ve got you covered. Read more about Lisk’s consensus algorithm and its benefits in the Lisk Academy.",
i18n/locales/en/common.json:  "What is Lisk Academy?": "What is Lisk Academy?",
src/assets/images/icons/arrow.svg:            <g id="Academy" transform="translate(553.000000, 486.593062)">
src/components/screens/delegates/delegates.js:      content: t('Want to know more? We’ve got you covered. Read more about Lisk’s consensus algorithm and its benefits in the Lisk Academy.'),
src/components/screens/help/help.js:              {this.props.t('Academy')}
src/components/shared/header/topBar/accountMenu/userAccount.js:          href={externalLinks.liskAcademy}
src/components/shared/header/topBar/accountMenu/userAccount.js:          <span>{t('Lisk Academy')}</span>
src/constants/externalLinks.js:  liskAcademy: 'https://lisk.io/academy',
src/constants/quickTips.js:    title: t('What is Lisk Academy?'),
src/constants/quickTips.js:      t('The Lisk Academy is an entirely free, unbiased and comprehensive educational platform about blockchain technology, containing something for everyone, regardless of what level of knowledge you are at.'),
src/constants/quickTips.js:      link: links.liskAcademy,

■■■■[slaweet:~/git/hub-review] $
$ git grep "Help Center"
app/src/menu.js:            label: i18n.t('Lisk Help Center'),
i18n/locales/de/common.json:  "Help Center": "Hilfebereich",
i18n/locales/de/common.json:  "Lisk Help Center": "Lisk Help Center",
i18n/locales/de/common.json:  "Visit Help Center": "Visit Help Center",
i18n/locales/en/common.json:  "Help Center": "Help Center",
i18n/locales/en/common.json:  "Lisk Help Center": "Lisk Help Center",
i18n/locales/en/common.json:  "Visit Help Center": "Visit Help Center",
src/components/screens/help/help.js:          <h2>{this.props.t('Help Center')}</h2>
src/components/screens/help/help.js:              {this.props.t('Help Center')}
src/components/screens/help/help.js:                {this.props.t('Visit Help Center')}

```

**Comment by osvaldovega** [comment]
totally agree, I will check those and remove it

**Comment by osvaldovega** [comment]
Thanks for the feedback Vit I checked with Yashar and he confirmed that we can remove all the links related to academy too, so I just update the PR please let me know what do you think.
The only pending thing are the icons/ilustrations.

File: src/constants/externalLinks.js [hunk 1]
```diff
@@ -1,24 +1,8 @@
...
   discord: 'https://discord.gg/CngsY6D',
-  explainBlockchain: 'https://lisk.io/academy/blockchain-basics/what-is-blockchain/eli5-blockchain',
-  gettingStarted: 'https://lisk.io/academy/welcome-to-the-lisk-academy/getting-started-with-lisk',
-  helpCenter: 'https://lisk.io/help-center',
   helpDeskChannel: 'https://lisk.chat/channel/help-desk',
-  howToStorePassphrase: 'https://lisk.io/academy/blockchain-business/cryptocurrencies/how-to-stay-secure-when-trading-cryptocurrencies',
-  isBlockchainSecure: 'https://lisk.io/academy/blockchain-basics/benefits-of-blockchain/blockchain-security',
-  ledgerNanoSHelp: 'https://support.ledger.com/hc/en-us/categories/115000820045-Ledger-Nano-S',
-  liskAcademy: 'https://lisk.io/academy',
   liskChat: 'https://lisk.chat/home',
```
Comment (yasharAyari) [inline src/constants/externalLinks.js]: Please remove the link of `lisk-chat` because we don't have it online anymore.

File: i18n/locales/en/common.json [hunk 2]
```diff
@@ -218,12 +211,8 @@
   "Language": "Language",
   "Last Forged Block": "Last Forged Block",
   "Later": "Later",
-  "Learn about blockchain with our comprehensive knowledge base.": "Learn about blockchain with our comprehensive knowledge base.",
-  "Learn more": "Learn more",
```
Comment (yasharAyari) [inline i18n/locales/en/common.json]: Make sure that you remove all unused translation strings from de.json.

**Comment by osvaldovega** [comment]
I removed lisk chat and and update the de file to remove all mentions about lisk chat, academy and help center from the file.

**Comment by Efefefef** [comment]
Test  🐛 💅
